brightStars_DM_sprint_Feb2020/BeyondRadius.ipynb

```diff
@@ -0,0 +1,422 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Study (false) detections around bright star masks\n",
+    "The radius of current HSC bright star masks, made by Andy Goulding, are selected by comparing the density of detections in increasingly large annuli around each bright stars with the average detection density. \n",
+    "\n",
+    "Still, one of the identified problems with the HSC Public Data Release 2 is that false detections sometimes occur around the edge of these masks. The aim of this notebook is thus to provide a plot of the number of detections as a function of increasing radius, around bright stars and beyond the edge of their mask."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Options\n",
+    "One might want to change the two cells below.\n",
+    "\n",
+    "The first one allows one to choose the path to the Butler repo we will read in, filter (note it only changes masks read in*, since we read detections from merged catalog), tracts, and binning options for the annuli.\n",
+    "\n",
+    "The rerun in `/project/mschmitz/hscRerun/mynobg` is a subset of the 25 central patches in tract 9697, and had local background subtraction turned off.\n",
+    "\n",
+    "Normally, we'd select bright objects to study from an outside catalog (Gaia). But for now, we are just reading in the actual mask region files, which means we have to jump through a few extra hoops.\n",
```

Comment:
Right - but the mask .reg files were definitely not intended to be used as a way to select bright stars.

The extra hoops are just specifically for these notebooks. What I meant is that it would perhaps be more natural to approach this by selecting bright stars using an external catalog, then computing the radial profile of detection density around these. What I have done is a bit of a retcon, reading bright star masks and using these as my list of objects of interest.

A specific example of what I meant by extra "hoops" is my hacky function to just parse the ds9 .reg file and extract the information we need: https://github.com/lsst-dm/drp-analysis-sprint-feb20/blob/16c25131f61a17e22fe4a35d7aeb17aa31a6ef36/brightStars_DM_sprint_Feb2020/helperFuncs.py#L7

Reply:
I think what shows in my writing and makes it confusing is I had bright star wings in mind when doing all this. While the plot/study can be used as a test of our bright star masks, once we have a way to subtract bright stars using an extended PSF model, we can _also_ use it as a test then - except we probably would not want to use a list of masks as a way to select which objects we look at.